scripts/redcap/purge_outdated.py

```diff
@@ -71,17 +75,21 @@ def parse_args(arg_input=None):
     return parser.parse_args()
     # return parser.parse_args(arg_input)
 
-
 def get_events(api, events=None, arm=None):
     # Based on arguments that were passed, output Redcap-compatible event names
     # for further consumption
-    pass
+
+    event_names = []        
+    for event in api.events:
+        if (event["unique_event_name"] in events):
+            if (event["arm_num"] == arm):
```

Comment:
This is only going to work if `arm` is always specified, but we should allow an `--event`-only specification

Reply:
...actually nevermind, I see that `arm` has a default of 1, so it should always work - ignore